src/components/Footer/Footer.js

```diff
@@ -2,7 +2,70 @@ import React from 'react';
 import './Footer.scss';
 
 const Footer = () => {
-  return <div>Footer 입니다.</div>;
+  return (
+    <div className="footerWrapper">
+      <div className="footerDiv">
```

Comment:
```suggestion
      <div className="footerDiv">
```
footerDiv가 어떤 의미인가요?